css/style_p2.css

```diff
@@ -0,0 +1,126 @@
+  ############################################################ p2
```

Comment:
It looks like `karma_form.html` and this file aren't being used any more; if so can you remove them?